### PR TITLE
donator gear And URF update

### PR DESCRIPTION
**Pull request by Flaksim**
thats all.

### CODE REVIEW

File: code/modules/halo/clothing/odst.dm [hunk 1]
```diff
@@ -184,14 +184,24 @@
 	item_state = "Odst Helmet Mann"
 	icon_state = "Odst Helmet Mann"
 
-
```
Comment (caelaislinn) [inline code/modules/halo/clothing/odst.dm:187]: Unnecessary line change

File: maps/insurrection/insurrection.dm [hunk 2]
```diff
@@ -12,8 +12,6 @@
 	#include "innie_base_overmap.dm"
 	#include "innie_base_map.dm"
 	#include "unsc_staging_areas.dm"
-	#include "oldinnie_base2.dmm"
-	#include "oldinnie_base1.dmm"
```
Comment (caelaislinn) [inline maps/insurrection/insurrection.dm]: Are these unused?

**Review by BDpuffy420 — APPROVED** [review]

File: code/modules/halo/clothing/odst.dm (hunk 1 above)
Comment (Flaksim) [reply code/modules/halo/clothing/odst.dm:187]: Yes, the extra blank line there was useless so I got rid of it.

File: maps/insurrection/insurrection.dm (hunk 2 above)
Comment (Flaksim) [reply maps/insurrection/insurrection.dm]: For the time being yes, that mode has issues I’m working on resolving.

**Comment by BDpuffy420** [comment]
!!! FAILURE !!! [MAP: Ladder Check]: One or more ladders are incorrectly setup.

**Comment by Flaksim** [comment]
Provided this last travis check passes this is done.

**Comment by caelaislinn** [comment]
You need to split this into a separate PR for the donator gear, and for the map changes. As it is this is difficult to review due to all of the changes thrown together but not properly documented, which is delaying the donator gear from getting merged.

**Comment by BDpuffy420** [comment]
passing

**Pull request merged**